controls/datagrid/aggregates/delegate-aggregate-descriptor.md

```diff
@@ -19,22 +19,32 @@ To set up the `DelegateAggregateDescriptor`, use the following properties:
 
 The following example uses the `DelegateAggregateDescriptor` and a custom implementation for a `SumIf` function which sums the values in a range that meet a certain criteria:
 
-**1.** First, create a class that inherits from the `IKeyLookup` interface. It will return the values of a Price property declared in our business model that is of type double.
+**1.** Create a class that inherits from the `IKeyLookup` interface. It will return the values of a `Price` property declared in our business model that is of type `double`.
```

Comment:
* Using present simple over other tenses is highly recommended by our Style Guide.
* Avoiding "our" and "we" is also highly recommended by our Style Guide.